### PR TITLE
Fixed scripts

### PR DESCRIPTION
**Pull request by UnknownHero99**

### CODE REVIEW

**Comment by UnknownHero99** [comment]
Added fix for broken CBS offer type

File: streamingscrapper/spiders/fxnetworks.py [hunk 1]
```diff
@@ -43,8 +43,8 @@ def parse_json(self, response):
         json_data = response.json()
         if json_data["show"]["type"] == "Movie":
             if "video" not in json_data:
-                self.logger.error("No video in json: " + str(json_data))
-
+                self.logger.warning("No video in json: " + str(json_data))
```
Comment (kodmarcel) [inline streamingscrapper/spiders/fxnetworks.py:46]: Change this logging into:
self.logger.info("Skipping unavaliable program " + response.url)

Comment (kodmarcel) [reply streamingscrapper/spiders/fxnetworks.py:46]: Also change version number!!

Comment (UnknownHero99) [reply streamingscrapper/spiders/fxnetworks.py]: changed

**Pull request merged**